using_images/db_images/mysql.adoc

```diff
@@ -160,6 +160,83 @@ the database:
 * *_/var/lib/mysql/data_* - This is the data directory where MySQL stores
 database files.
 
+=== Changing passwords
+
+Since passwords are part of the image configuration, the only supported method
+to change passwords for the database user (`*MYSQL_USER*`) and root user is by
+changing the environment variables `*MYSQL_PASSWORD*` and
+`*MYSQL_ROOT_PASSWORD*`, respectively.
+
+Whenever `*MYSQL_ROOT_PASSWORD*` is set, it enables remote access for the root
+user with the given password, and whenever it is unset, remote access for the
+root user is disabled.
+This doesn't affect the regular user `*MYSQL_USER*` who always has remote
+access.
+This also does not affect local access by the root user, who can always login
+without password in *localhost*.
+
+Changing database passwords through SQL statements or any way other than through
+the environment variables aforementioned will cause a mismatch between the
+values stored in the variables and the actual passwords. Whenever a database
+container starts it will reset the passwords to the values stored in the
+environment variables.
+
+You can change these passwords with:
+
+----
+$ oc env dc mysql \
+  MYSQL_PASSWORD=NewPassword \
+  MYSQL_ROOT_PASSWORD=NewAdminPassword
+----
+
+This will trigger the redeployment of the database server if you have a
+link:../../dev_guide/deployments.html#config-change-trigger[configuration change trigger].
+Otherwise, you need to manually start a new deployment in order to apply the
+password changes.
+
+Note that you can change one password but not the other by simply omitting one
+of the arguments to `oc env` above.
+
+You can verify that the new password is in effect with:
+
+----
+$ oc get pods
+...
+$ oc exec <DB_PODNAME> -- bash -c 'mysql -u $MYSQL_USER -pNewPassword -h mysql $MYSQL_DATABASE -te "SELECT * FROM (SELECT database()) db CROSS JOIN (SELECT user()) u"'
+----
+
+Replace *<DB_PODNAME>* with the name the running MySQL pod.
```

Comment:
Excellent catch, thanks! Fixed also the missing prepositions on similar lines for MongoDB and PostgreSQL.